linux/helper_centos.sh

```diff
@@ -10,6 +10,10 @@ add_repo() {
     fi
 
     wget -nv -O /etc/yum.repos.d/newrelic-infra.repo "$repo"
+    # prod .repo file points to the cache url, we replace it to point to the bucket url
+    rs='s/https:\/\/download\.newrelic\.com/http:\/\/nr-downloads-main.s3-website-us-east-1.amazonaws.com/'
+    sed -i $rs /etc/yum.repos.d/newrelic-infra.repo
```

Comment:
Sed expression can be simplified a bit by using other character as delimiter :)
```suggestion
    rs='s|https://download.newrelic.com|http://nr-downloads-main.s3-website-us-east-1.amazonaws.com|'
    sed -i "$rs" /etc/yum.repos.d/newrelic-infra.repo
```